### PR TITLE
Puts the new and improved quiz up

### LINKED ISSUE #1063: Update quiz page
- [ ] h1 tag
- [ ] include opening partial
- [ ] segment the email capture

### PR DESCRIPTION
**Pull request by alexandergimson**
Closes #1063

### CODE REVIEW

**Comment by alexandergimson** [comment]
@nikeshashar pull this down and see what you think

File: source/quiz.html.haml [hunk 1]
```diff
@@ -1,12 +1,24 @@
 - description "Makers Academy is London's leading Web Developer Bootcamp. Learn to code in 12 Weeks by enrolling on one of our programming / web development courses! Learn Ruby, JavaScript and the best software practices."
```
Comment (nikeshashar) [inline source/quiz.html.haml]: we need a title

File: source/quiz.html.haml [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+  (function(){var qs,js,q,s,d=document,gi=d.getElementById,ce=d.createElement,gt=d.getElementsByTagName,id='typef_orm',b='https://s3-eu-west-1.amazonaws.com/share.typeform.com/';if(!gi.call(d,id)){js=ce.call(d,'script');js.id=id;js.src=b+'widget.js';q=gt.call(d,'script')[0];q.parentNode.insertBefore(js,q)}})();
 
-%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/dan-and-students-overhead-70.jpg")});"}
+%section.hero-splash
   .container
     .centered-row
-      %P Makers Academy is Europe's leading Web Developer bootcamp. Get a free copy of our Learn Ruby In 15 Minutes e-book by signing up to our newsletter.
+      %h2 How did you do?
+      %P Did you get 9 out of 9? Makers Academy is Europe's leading Web Developer bootcamp. Get a free copy of our Learn Ruby In 15 Minutes e-book by signing up to our newsletter.
     = partial :"email-capture"
```
Comment (nikeshashar) [inline source/quiz.html.haml]: we need a segment for this in customer.io

File: source/quiz.html.haml [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/newrow-of-women.jpg")});"}
+  .container
+    .centered-row
+      %h4.leader Take our
+      %h1 Web developer quiz
+      %p Think you know your HTML from your CSS? Your JavaScript from your Java? Or even your Mac from your PC?
+      %p Take our web developer quiz and find out if you've got what it takes to become a web developer!
+
+%section.typeform
+  .typeform-widget{style: "height:980px;", "data-text" => "Hire Graduates", "data-url" => "https://makersacademy.typeform.com/to/EOkP5P"}
```
Comment (nikeshashar) [inline source/quiz.html.haml]: update `data-text`

**Pull request merged**